test/unit/notices_processCBNotice.js

```diff
@@ -0,0 +1,452 @@
+
+                // this feature does not seem to make sense
+                var at = 'location';
+                var parseLocationMock = sinon.spy(function() {
+                    return {
+                        lat: lat,
+                        lon: long,
+                        x: x,
+                        y: y
+                    };
+                });
+                notices.__with__({
+                    'uuid.v1': uuidMock,
+                    'Date.now': dateNowMock,
+                    'parseLocation': parseLocationMock
+                })(function () {
+                    noticeExample.contextResponses[0].contextElement.attributes[0].metadatas = [{
+                        'name':  at,
+                        'value': locValue,
+                        'type': locType
+                    }];
+                    var noticeResult = processCBNotice(service, subservice, noticeExample, 0);
+                    expect(noticeResult.noticeId).to.equal(mockedUid);
+                    expect(noticeResult.noticeTS).to.equal(mockedDateMilis);
+                    expect(noticeResult.id).to.equal(id);
+                    expect(noticeResult.type).to.equal(type);
+                    expect(noticeResult.subservice).to.equal(subservice);
+                    expect(noticeResult.service).to.equal(service);
+                    expect(noticeResult.isPattern).to.equal('false'); // why not boolean??
+                    expect(noticeResult[attrKey + '__type']).to.equal(attrType);
+                    expect(noticeResult[attrKey]).to.equal(attrValue);
+                    expect(noticeResult[attrKey + '__metadata__' + at]).to.equal(locValue);
+                    expect(noticeResult[attrKey + '__metadata__' + at + '__type']).to.equal(locType);
+
+                    // This attributes not should be metadata???
+                    expect(noticeResult[attrKey + '__lat']).to.equal(lat);
+                    expect(noticeResult[attrKey + '__lon']).to.equal(long);
+                    expect(noticeResult[attrKey + '__x']).to.equal(x);
+                    expect(noticeResult[attrKey + '__y']).to.equal(y);
+                    //expect(noticeResult[attrKey + '__metadata__' + at + '__lat']).to.equal(lat);
+                    //expect(noticeResult[attrKey + '__metadata__' + at + '__lon']).to.equal(long);
+                    //expect(noticeResult[attrKey + '__metadata__' + at + '__x']).to.equal(x);
+                    //expect(noticeResult[attrKey + '__metadata__' + at + '__y']).to.equal(y);
+
+                    // Why call parselocation with the attribute value and not with location metadata attribute?
+                    parseLocationMock.should.have.been.calledWith(attrValue);
+                    parseLocationMock.should.be.calledOnce;
+                    done();
+                });
+            }
+        );
+        // why NGSIv1 dont parse metadata geo:points? only parseocation when 'name' is 'location'.
```

Comment:
This is a real question? :)

Answer: `geo:...` attribute types are part of the NGSIv2 specification. It's usage in NGSIv1 is CB is undefied (not sure right now... maybe there is some degree of support, but I don't remember pretty well right now).

Reply:
This block of code should be removed? We don't like to leave dead code behind us, do we? ;)

Reply:
Ooops jejeje. This is not a real question, it's just my internal questions. I do not have much experience with NGSIv1 :)

Fixed in 24ade02